05/Task05.js

```diff
@@ -19,11 +21,25 @@ const Task04 = () => {
                 </RBCard>
             </Col>
             <Col>
-                Card!
+                <Card>
+                    <CardImg src="https://picsum.photos/100/80"></CardImg>
+                    <CardBody>
+                        <CardTitle>Card Title</CardTitle>
+                        <CardText>
+                            Some quick example text to build on the card title and make up the bulk of
+                            the card's content.
+                        </CardText>
+                        <Button variant="success">Go somewhere</Button> 
+                    </CardBody>
+                </Card>
             </Col>
         </Row>
     )
 }
 
-export default Task04;
+export default Task05;
+
+// <Button variant="success" size='small'>Go somewhere</Button>  - mogę tutaj dodać jeszcze style? 
```

Comment:
<Button> to normalny komponent Reactowy więc musisz ten props przekazać do StyledComponent:
```
const Button = props => {
    const {variant, size, active, disabled, children, style} = props;
    return (
        <StyledButton variant={variant} size={size} active={active} disabled={disabled} style={style}>{children} </StyledButton>
    )
}
```

Pewnym wygodniejszym rozwiązaniem, ale nie koniecznie bezpiecznym (zbyt duża elastyczność - można przekazać wszystko) to coś takiego (tylko warto usunać `children` z `props`):
```
<StyledButton {...props}>{children} </StyledButton>
```